waspc/src/Wasp/Parser/Server.hs

```diff
@@ -23,8 +23,8 @@ server = do
           fromMaybe (error "Server js function is missing.") (getSetupJsFunctionFromProps props)
       }
 
-data Property
-  = SetupJsFunction !Wasp.JsImport.JsImport
+newtype Property
+  = SetupJsFunction Wasp.JsImport.JsImport
```

Comment:
Why did `!` get removed? I am actually pretty sure that I saw `stan` saying we should use `!` at places like these?

Reply:
With `newtype` the `!` is not permitted.